sorc/nst_tf_chg.fd/CMakeLists.txt

```diff
@@ -16,9 +16,9 @@ set(exe_name nst_tf_chg)
 add_executable(${exe_name} ${fortran_src})
 target_link_libraries(
   ${exe_name}
-  nemsio
-  w3nco_d
-  bacio_4
+  nemsio::nemsio
+  w3nco::w3nco_d
+  bacio::bacio_4
```

Comment:
All three nemsio_* utilities depend directly only on nemsio library. w3nco and bacio can be removed.

Reply:
done.